composer.json

```diff
@@ -15,14 +15,13 @@
     ],
     "require": {
         "php": ">=5.4.0",
-        "illuminate/support": "~4.2|~5.0"
+        "illuminate/support": "~5.0"
```

Comment:
I can update this, but note that this PR is 2 months old, long before 5.1 was released. Also, isn't Laravel using semver? Considering how long it's taken for a code review here, making it 5.1.x will be great until 5.2 is released.

Also, are you sure 5.0.x isn't supported anymore: http://laravel.com/docs/5.1/releases ?
>  For general releases, bug fixes are provided for 6 months and security fixes are provided for 1 year.